Automattic/newspack-plugin
# feat(google-oauth): handle invalid access token; add GA scopes

### All Submissions:






### Changes proposed in this Pull Request:



A UX improvement to the OAuth flow.

### How to test the changes in this Pull Request:

1. On `master`,
1. Connect your site to Google account via Connections wizard
2. [Revoke access](https://myaccount.google.com/permissions) for the app
3. Reload Connections wizard, observe no notice
4. Switch to this branch, observe a notice about invalid credentials added
5. Revoke access again, and complete the setup but do _not_ check the GAM or GA scopes*
6. Observe a warning about insufficient scopes granted
7. Complete to flow again, granting all scopes
8. Observe no errors and connected state of the UI

\* You'll have to use a Google account not associated with the OAuth app (as a developer) for this flow to be available:

<img width="590" alt="image" src="https://user-images.githubusercontent.com/7383192/141329192-c1fbeb00-e831-4243-9496-a43259754461.png">


### Other information:
